docs/tutorials/multimodal/object_detection/prepare_coco17.md

```diff
@@ -0,0 +1,133 @@
+# AutoMM Detection - Prepare COCO2017 Dataset
+:label:`sec_automm_detection_prepare_coco17`
+
+[COCO](https://cocodataset.org/#home) is a large-scale object detection, segmentation, and captioning dataset. 
+For detection, the most commonly used version is COCO17 with the largest number of training data.
+There are 80 classes, 123,287 images, 886,284 instances, and the median image ratio is 640 x 480.
+This tutorial will walk through the steps of preparing this dataset for Autogluon MultiModalPredictor.
+
+You need 42.7 GB disk space to download and extract this dataset. SSD is preferred over HDD because of its better performance.
+The total time to prepare the dataset depends on your Internet speed and disk performance. For example, it often takes 20 min on AWS EC2 with EBS.
+
+COCO has an [official download page](https://cocodataset.org/#download), 
+but it's always easier to perform a one-step setup. 
+You can use the bash script for downloading coco17: [download_coco17.sh](https://github.com/awslabs/autogluon/blob/master/examples/automm/object_detection/download_coco17.sh),
+or use the python script that can download all datasets mentioned in our tutorials: [prepare_detection_dataset.py](https://github.com/awslabs/autogluon/blob/master/examples/automm/object_detection/prepare_detection_dataset.sh).
+
+### Download with Python Script
+
+The python script does not show progress bar, but is promised to work on all major platforms.
+If you are working on a Unix system and needs a progress bar, try the bash script!
+
+You could either extract it in coco17 folder under current directory by running:
+
+```
+python3 prepare_detection_dataset.py --dataset_name coco2017
+```
+
+or extract it in coco17 folder under a provided output path:
+
+```
+python3 prepare_detection_dataset.py --dataset_name coco2017 --output_path ~/data
+```
+
+or make it shorter:
+
+```
+python3 prepare_detection_dataset.py -d coco17 -o ~/data
+```
+
+### Download with Bash Script
+
+You could either extract it in coco17 folder under current directory by running:
+
+```
+bash download_coco17.sh
+```
+
+or extract it in coco17 folder under a provided output path:
+
+```
+bash download_coco17.sh ~/data
+```
+
+The command line output will show the progress bar:
+
+```
+extract data in ./coco17
+--2022-11-02 20:19:49--  http://images.cocodataset.org/zips/train2017.zip
+Resolving images.cocodataset.org (images.cocodataset.org)... 52.217.18.68
+Connecting to images.cocodataset.org (images.cocodataset.org)|52.217.18.68|:80... connected.
+HTTP request sent, awaiting response... 200 OK
+Length: 19336861798 (18G) [application/zip]
+Saving to: ‘train2017.zip’
+
+train2017.zip                          100%[=========================================================================>]  18.01G  27.0MB/s    in 7m 29s  
```

Comment:
outputs should be removed unless it is very important. the compilation process will produce these outputs

Reply:
I disabled running the code script and paste the outputs to reduce CI pressure.

Reply:
Codes in tutorial are not executed so the output is pasted here to be consistent with other tutorials.